user-manual/ingest/manual-normalization.rst

```diff
@@ -71,7 +71,7 @@ normalized according the regular Archivematica workflow.
 
 2. Select "Normalize manually" from the dropdown menu.
 
-3. In your file browser, open /archivematica/src/MCPServer/sharedDirectoryStructure/watchedDirectories/approveNormalization/preservation/yourTransfer123abc/objects/
+3. In your file browser, go to your pipelines Processing Location, and find the approveNormalization watched folder.  The default location is at /var/archivematica/sahredDirectory/watchedDirectories/approveNormalization/preservation/yourTransfer123abc/objects/ .  You may need to look at your storage service configuration to confirm the Processing Location defined for your pipeline.
```

Comment:
Could you use the `:file:` inline markup? e.g.

```rst
3. In your file browser, go to your pipelines Processing Location, and find the approveNormalization watched folder.  The default location is at :file:`/var/archivematica/sahredDirectory/watchedDirectories/approveNormalization/preservation/yourTransfer123abc/objects/`.  You may need to look at your storage service configuration to confirm the Processing Location defined for your pipeline.
```

More info here: http://www.sphinx-doc.org/en/1.4.8/markup/inline.html#role-file. It's handy because it generates different HTML output, e.g.:

```html
<code class="file docutils literal"><span class="pre">/var/archivematica/sahredDirectory/watchedDirectories/approveNormalization/preservation/yourTransfer123abc/objects/</span></code>
```

And we have pre-defined stylesheets for that kind of stuff.